torch/csrc/Module.cpp

```diff
@@ -353,6 +353,7 @@ PyObject *THPModule_setDeterministicCuDNN(PyObject *_unused, PyObject *arg)
 {
   THPUtils_assert(PyBool_Check(arg), "set_deterministic_cudnn expects a bool, "
           "but got %s", THPUtils_typename(arg));
+  THPUtils_assert(!(arg == Py_True && at::globalContext().benchmarkCuDNN() == true), "Cannot set torch.backends.cudnn.benchmark=True and torch.backends.cudnn.deterministic=True at the same time because they have opposite behaviors.");
```

Comment:
The line is too long I suppose.

Reply:
And isn't it too harsh to issue an error instead of a warning as discussed in #6351 ?

Reply:
I see, I will change it to be a warning only